bridge-api/definitions/constraints/integer_constraints.yml

```diff
@@ -2,7 +2,7 @@ description: |
     This type can be used to collect a number, or with a unit, it can be used to collect 
     a measure or a duration (e.g. "3 hours", or "180 pounds").
 
-    **UI hints:** numberfield, slider.
+    **UI hints:** numberfield, slider, select.
 required: 
     - dataType
 allOf:
```

Comment:
For all three classes I would also add comment that minValue/maxValue properties are required if uiHint is slider or select.